Makefile

```diff
@@ -99,6 +99,11 @@ config: FORCE
 		&& echo 'CONFIG_GLUON_RELEASE="$(GLUON_RELEASE)"' \
 		&& echo 'CONFIG_GLUON_SITEDIR="$(GLUON_SITEDIR)"' \
 		&& echo 'CONFIG_GLUON_BRANCH="$(GLUON_BRANCH)"' \
+		$(if $(DEBUG),&& echo 'CONFIG_DEBUG=y' \
+			&& echo 'CONFIG_NO_STRIP=y' \
+			&& echo '# CONFIG_USE_STRIP is not set' \
+			&& echo '# CONFIG_USE_SSTRIP is not set' \
```

Comment:
Have you checked that all these 3 lines are necessary, and just setting `CONFIG_NO_STRIP=y` doesn't have the same effect?